neutrino.go

```diff
@@ -86,6 +86,22 @@ var (
 	DefaultBlockCacheSize uint64 = 4096 * 10 * 1000 // 40 MB
 )
 
+// SideLoadOpt defines the config required to sideload headers into the DB.
+type SideLoadOpt struct {
```

Comment:
See comment about using proper struct tags if meant to be read from the command line.

Reply:
I don't quite get this. This is added as part of the neutrino config.
https://github.com/lightninglabs/neutrino/blob/e13c95fa3e76fd1883a8b2d0db0ba96a53b9b4c2/neutrino.go#L638C1-L638C22

Reply:
So if neutrino just has the initialized side loader passed in, then it doesn't need to worry about where the files actually live, as it uses the main interface. We'd then add an arg to `lnd` that would read the headers from disk or w/e, make a concrete implementation of the intefrace, then pass it into `neutrino`.

Reply:
Please confirm if this has been addressed in the newest revision of the PR with the `io.readerSeeker` included as a field.